src/main/java/org/assertj/core/error/ShouldBeEqualNormalizingPunctuation.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.error;
+
+/**
+ * Creates an error message indicating that an assertion that verifies that two {@code CharSequence}s are equal,
+ * after the punctuations of both strings has been normalized, failed.
+ *
+ * Created by harisha talanki on 2/29/20
+ */
+public class ShouldBeEqualNormalizingPunctuation extends BasicErrorMessageFactory {
```

Comment:
add a test for this class (https://github.com/joel-costigliola/assertj-core/blob/master/src/test/java/org/assertj/core/error/ShouldBeBetween_create_Test.java  is a good example of one)